includes/utils.php

```diff
@@ -435,6 +435,7 @@ function excluded_meta() {
 function prepare_meta( $post_id ) {
 	update_postmeta_cache( array( $post_id ) );
 	$meta          = get_post_meta( $post_id );
+	$meta          = is_array( $meta ) ? $meta : array();
```

Comment:
Here are the chances of a PHP 8+ incompatibility warning: `Automatic conversion of false to array is deprecated`.

Reply:
@faisal-alvi I've added early return in case meta is false.